packages/docusaurus/lib/commands/start.js

```diff
@@ -55,12 +54,7 @@ module.exports = async function start(siteDir, cliOptions = {}) {
       ),
     );
     const fsWatcher = chokidar.watch(
-      [
-        ...pluginPaths,
-        `../${docsRelativeDir}/**/*.md`,
-        loadConfig.configFileName,
-        'sidebars.json',
-      ],
+      [...pluginPaths, loadConfig.configFileName, 'sidebars.json'],
```

Comment:
I think `Sidebars.json` is supposed to be inside docs plugin path to watch ?

Reply:
Yes. I'll move the sidebar stuff in a future PR and let sidebar be a plugin option.